CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## CHANGE LOG.
 
+### July,19, 2022
```

Comment:
//todo : need to update this  date once ready to publish

Reply:
@root-ansh-ct Finish all TODOs